07-Customizing-AzSK-for-your-Org/Readme.md

```diff
@@ -460,7 +460,35 @@ image below:
 This change will be effective across your organization immediately. Anyone running AzSK commands (in fresh PS sessions)
 should see the new message. 
 
-##### b) Changing a control setting for specific controls 
+##### b) Customize output folder for my org
+Normally, when we run GSS for our org policy its security scan results goes to an folder 
+`%LOCALAPPDATA%\Microsoft\AzSKLogs\Sub_[yourSubscriptionName]E.g.`
+C:\Users\UserName\AppData\Local\Microsoft\AzSKLogs\Sub_[yourSubscriptionName]\20170331_142819. We can customize this location also so that our scan result log can be collected at a particular location given by you.
```

Comment:
This location can be customized in an org policy so that the scan results can be collected at a particular location on org policy users machine.

Reply:
Made changes

Reply:
Done